kvrocks.conf

```diff
@@ -745,10 +745,61 @@ rocksdb.write_options.low_pri no
 
 # If enabled, this writebatch will maintain the last insert positions of each
 # memtable as hints in concurrent write. It can improve write performance
-# in concurrent writes if keys in one writebatch are sequential. 
+# in concurrent writes if keys in one writebatch are sequential.
 #
 # Default: no
 rocksdb.write_options.memtable_insert_hint_per_batch no
 
+# A list of paths where SST files can be put into, with its target size.
+# Newer data is placed into paths specified earlier in the vector while
+# older data gradually moves to paths specified later in the vector.
+#
+# For example, you have a flash device with 10GB allocated for the DB,
+# as well as a hard drive of 2TB, you should config it to be:
+#   [{"/flash_path", 10GB}, {"/hard_drive", 2TB}]
+#
+# The system will try to guarantee data under each path is close to but
+# not larger than the target size. But current and future file sizes used
+# by determining where to place a file are based on best-effort estimation,
+# which means there is a chance that the actual size under the directory
+# is slightly more than target size under some workloads. User should give
+# some buffer room for those cases.
+#
+# If none of the paths has sufficient room to place a file, the file will
+# be placed to the last path anyway, despite to the target size.
+#
+# Placing newer data to earlier paths is also best-efforts. User should
+# expect user files to be placed in higher levels in some extreme cases.
+#
+# If left empty, only one path will be used, which is db_name passed when
```

Comment:
What option do these lines describe (with Default: empty)?

Reply:
These lines describe the behavior of the "datadir" option in a database configuration file.

By default, if "datadir" is left empty, only one path will be used for storing the database files. However, if multiple paths are specified, the database will attempt to place newer data in the earlier paths.

The lines also state that this behavior is "best-efforts," meaning that the database will try its best to follow this rule but may not always be able to do so. In some extreme cases, user files may be placed in higher levels, which is something the user should be aware of.

Overall, these lines describe how the "datadir" option works and what behavior the user can expect if multiple paths are specified.